src/components/Person/Person.jsx

```diff
@@ -1 +1,30 @@
-// Write code here
+import React from 'react';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => (
+  // eslint-disable-next-line
+  <>
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+
+      {age && (
+        <p className="Person__age">
+          {`I am ${age}`}
+        </p>
+      )}
+
+      <p className="Person__partner">
+        {isMarried
+          ? `My ${sex === 'm' ? 'wife' : 'husband'}'s name is ${partnerName}`
```

Comment:
I'd move it to the variable, too much logic in JSX is bad

Reply:
there is only 1 ternary operator, I can't see reason to create variable. After all, this block of code is not reused